src/Components/TwinPropertySearchDropdown/TwinPropertySearchDropdown.tsx

```diff
@@ -197,6 +209,13 @@ const TwinPropertySearchDropdown: React.FC<ITwinPropertySearchDropdownProps> = (
         );
     };
 
+    // imperative handle to expose modifying the state
+    useImperativeHandle(ref, () => ({
+        updateValue: (newValue: string) => {
```

Comment:
I'm thinking we probably need to create an option for the value as well right? what happens if you set it to a value not found in the list? Ideally wouldn't happen but it should be resilient to that and add the item if not already there. I think you can just set the `selectedOption` since we add the selected option as an item if it's not in the list.

Reply:
Should be fixed, please take a look at it to confirm